packages/grid_client/tests/modules/tft.test.ts

```diff
@@ -0,0 +1,98 @@
+import Decimal from "decimal.js";
+
+import { type GridClient } from "../../src";
+import { TFT } from "../../src/modules/tft";
+import { config, getClient } from "../client_loader";
+
+jest.setTimeout(300000);
+
+const mock_price = jest.fn().mockResolvedValue(0.2);
+
+let grid: GridClient;
+let tft: TFT;
+
+beforeAll(async () => {
+  grid = await getClient();
+  const substrate_url = grid.getDefaultUrls(config.network).substrate;
+
+  tft = new TFT(substrate_url);
+
+  tft.price = mock_price;
+});
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+afterAll(async () => {
+  await grid.disconnect();
+});
+
+describe("Testing TFT module", () => {
+  test("Create TFT instance in all networks.", () => {
+    // Test Data
+    const SUBSTRATE_DEV = "wss://tfchain.dev.grid.tf/ws";
+    const SUBSTRATE_QA = "wss://tfchain.qa.grid.tf/ws";
+    const SUBSTRATE_TEST = "wss://tfchain.test.grid.tf/ws";
+    const SUBSTRATE_MAIN = "wss://tfchain.grid.tf/ws";
```

Comment:
if you already created the enum, you can use it here instead of re-creating the values

Reply:
Removed it as it's not used anywhere.